index.js

```diff
@@ -6,10 +6,16 @@ var VersionChecker = require('ember-cli-version-checker');
 module.exports = {
   name: 'ember-let',
 
+  init() {
+    this._super.init && this._super.init.apply(this, arguments);
+
+    var checker = new VersionChecker(this);
+    this._emberDep = checker.forEmber();
```

Comment:
Looks like this is undefined in tests? https://travis-ci.org/thefrontside/ember-let/jobs/253925527#L494

Reply:
It looks like this isn't working because init() on addon calls `setupPreprocessorRegistry()` before we set `this._emberDep`.

Reply:
Yeah I debugged it a little bit last night but ran into version checker issues and gave up :p

Reply:
#32 deals with that right?

Reply:
It does not. In this PR it's bumped: https://github.com/thefrontside/ember-let/pull/43/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R52

I even tried to go to 2.x and still got the same error.